bookstore-vapor/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:jammy
+
+# Make sure all system packages are up to date, and install only essential packages.
+RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true \
+    && apt-get -q update \
+    && apt-get -q install -y \
+      ca-certificates \
+      tzdata \
+      libc6 \
+# If your app or its dependencies import FoundationNetworking, also install `libcurl4`.
+      libcurl4 \
+# If your app or its dependencies import FoundationXML, also install `libxml2`.
+      # libxml2 \
+    && rm -r /var/lib/apt/lists/*
+
+# Create a vapor user and group with /app as its home directory
+RUN useradd --user-group --create-home --system --skel /dev/null --home-dir /app vapor
+
+# Switch to the new home directory
+WORKDIR /app
+
+# Copy built executable and any staged resources from builder
+COPY ./staging /app
```

Comment:
Similar comment as in the other file. I would suggest rewriting this file to a Multi Stage Dockerfile